src/dual_sets.jl

```diff
@@ -34,4 +34,8 @@ end
 
 function dual_set(s::MOI.RotatedSecondOrderCone)
     return MOI.RotatedSecondOrderCone(MOI.dimension(s))
+end
+
+function dual_set(s::MOI.PositiveSemidefiniteConeTriangle)
+    return MOI.PositiveSemidefiniteConeTriangle(s.side_dimension) # Change to MOI.side_dimension in MOI 0.9.0
```

Comment:
Maybe just `copy(s)` ? In fact `s` is not mutable so `copy` is a no-op but you should no rely on that.

Reply:
Indeed it is better but I don't understand why
```
julia> s = MOI.PositiveSemidefiniteConeTriangle(5)
MathOptInterface.PositiveSemidefiniteConeTriangle(5)

julia> @benchmark MOI.PositiveSemidefiniteConeTriangle($s.side_dimension)
BenchmarkTools.Trial: 
  memory estimate:  0 bytes
  allocs estimate:  0
  --------------
  minimum time:     1.549 ns (0.00% GC)
  median time:      1.656 ns (0.00% GC)
  mean time:        1.728 ns (0.00% GC)
  maximum time:     54.250 ns (0.00% GC)
  --------------
  samples:          10000
  evals/sample:     1000

julia> @benchmark copy($s)
BenchmarkTools.Trial: 
  memory estimate:  0 bytes
  allocs estimate:  0
  --------------
  minimum time:     0.032 ns (0.00% GC)
  median time:      0.035 ns (0.00% GC)
  mean time:        0.040 ns (0.00% GC)
  maximum time:     43.367 ns (0.00% GC)
  --------------
  samples:          10000
  evals/sample:     1000
```

Reply:
`copy(s) = s` in MOI since we exploit that it is immutable. We can exploit it in MOI but you cannot do it outside of the package

Reply:
hmm I understood that because it is immutable it performs better but I don`t understand what are the problems of using it inside Dualization.